lib/version_utils.pm

```diff
@@ -652,3 +653,42 @@ sub has_test_issues() {
     }
     return 0;
 }
+
+=head2 package_version_cmp
+
+Compare two SUSE-style version strings. Returns an integer that is less than,
+equal to, or greater than zero if the first argument is less than, equal to,
+or greater than the second one, respectively.
+
+=cut
+
+sub package_version_cmp {
```

Comment:
Mind adding tests for this specifically? https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/t/01_version_utils.t

Reply:
Unit test added.